### PR TITLE
Fix/run test in docker

### PR DESCRIPTION
**Pull request by FredrikMorstad**
### usage :book: 
  - The script execute pytest command inside a running containe, so all flags addded when running the script will be added to the command. `python3 pytest_docker.py - s` will run `pytest -s ` inside the container
### Setup :mag:  
- The script uses docker SDK for python to create a container running mongodb
- After mongodb is running the script executes the pytest command on tdctl_api (depends on running api) 
### Discussion:
  - This script can be integrated in pytest, so that a user can for example run `pytest --docker` and the script get executed as a part of pytest
    - The reason this is not implemented is that it introduces another dependency which means that a user has to install both docker sdk and pytest. Any comments for or against this?

### CODE REVIEW

**Comment by erlingdevold** [comment]
Why is there a pipfile.lock file in the commit? 😀

**Comment by FredrikMorstad** [comment]
Lock files should be added

https://stackoverflow.com/questions/46278288/git-should-pipfile-lock-be-committed-to-version-control

**Comment by erlingdevold** [comment]
> My understanding is that lock files should be added🔐
> 
> https://stackoverflow.com/questions/46278288/git-should-pipfile-lock-be-committed-to-version-control

Yes, that is the correct answer, fredrik!

File: README.md [hunk 1]
```diff
@@ -3,28 +3,43 @@
...
   * To run the API locally use the `manage.py` script, which will run a local instance of the API:  ```./manage.py``` 
   * To run mongodb locally you must be inside the db folder, and then run `mongod -f db.yml`. `db.yml` uses relative paths, and the relative paths are relative from where you run the command from and not from the file itself.
   * The HTTP server should by default be run on [localhost:5000](http://localhost:5000/)
   * Please note that datasources(or equivalent mockup code) used by the API must be available for all functionality to work.
 
 #### 2.2 Docker
-  * To use docker, you can run `docker-compose -f .docker/docker-compose.development.yml`. This will launch both the backend and mongodb inside two separate containers.
+  * To use docker, you can run: 
+    1. `docker-compose -f .docker/docker-compose.development.yml build`
+    2. `docker-compose -f .docker/docker-compose.development.yml up`.
```
Comment (erlingdevold) [inline README.md:28]: Remove the punctuation on the second line, when you try to copy the command, you also get the punctuation :((

**Review by erlingdevold — APPROVED** [review]
`
====================== 27 passed, 212 warnings in 12.91s =======================   
`
LGTM

File: README.md [hunk 2]
```diff
@@ -3,28 +3,43 @@
...
-:information_source: Make sure the mongodb server is running. To start the mongo server  run `mongod -f db.yml` in the db folder
-  * To run the tests run the following command `pytest -rx`
+### 3.1 Running tests
+**:information_source: Make sure the mongodb server is running. To start the mongo server  run `mongod -f db.yml` in the db folder**
+  * To run all tests: `pytest -rx`
   * To run a specific test: `pytest -rx <path to test file>`
+
+See [pytest documentation](https://docs.pytest.org/en/7.2.x/) for more information
+### 3.2 Running tests in docker
+**To perfore tests when using docker, run the `pytest_docker.py` file.**
```
Comment (Sagensagen) [inline README.md:41]: Might be ideal to define where to execute these commands as people as my self tried to run everything from within the container it self initially.

**Review by Sagensagen — APPROVED** [review]
Looks good, Gj :fire:

**Pull request merged**